model/src/main/java/org/projectnessie/model/IcebergTable.java

```diff
@@ -47,19 +47,19 @@
     title = "Iceberg table global state",
     description =
         "Represents the global state of an Iceberg table in Nessie. An Iceberg table is globally "
-            + "identified via its unique 'Contents.id'.\n"
+            + "identified via its unique 'Content.id'.\n"
```

Comment:
nit: just `content ID` perhaps? What's the meaning of the `.` there? It does not look like a property path :thinking:

Reply:
not sure, there are also a few other places that refer to it like that. Would probably make sense to update this in a separate PR if we really want it